package.json

```
@@ -20,10 +21,10 @@
     "type-scale"
   ],
   "engines": {
-    "node": ">=12"
+    "node": ">=18"
   },
   "scripts": {
-    "test": "jest",
+    "test": "node --experimental-vm-modules ./node_modules/.bin/jest",
```

[suggest] To avoid the following `SyntaxError` on Windows, changing the *bin* script seems good:

```
> node --experimental-vm-modules ./node_modules/.bin/jest

D:\a\stylelint-scales\stylelint-scales\node_modules\.bin\jest:2
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
          ^^^^^^^

SyntaxError: missing ) after argument list
```

https://github.com/jeddy3/stylelint-scales/actions/runs/6884058566/job/18725831118?pr=94#step:6:7

```suggestion
    "test": "node --experimental-vm-modules ./node_modules/jest/bin/jest.js",
```

Scripts in `node_modules/.bin/` seem not to be a JavaScript file.

---

Great catch.